File: Ass7/Ass7P2.java

import java.util.Scanner;

class Number
{
    int DollarToINR(int iNo)
    {
        int iconvert = 0;

        iconvert = iNo * 70;

        return iconvert;
    }
}

class Ass7P2
{
    public static void main(String A[])
    {   
        Scanner sobj = new Scanner(System.in);
        int ivalue = 0,iret = 0;
        
        System.out.println("Enter Number :-");
        ivalue = sobj.nextInt();

        Number nobj= new Number();

        iret = nobj.DollarToINR(ivalue);

        System.out.println("Value in INR is :- "+iret);


    }
}

File: Ass7/Ass7P2.c
#include<stdio.h>

int DollarToInr(int iNo)
{
    int iconvert = 0;

    iconvert = iNo * 70;

    return iconvert;
   
}
int main()
{
    int ivalue = 0,iret = 0;

    printf("Enter Numer:- \n");
    scanf("%d",&ivalue);

    iret=DollarToInr(ivalue);
    printf("Value in INR is :- %d\n",iret);

    return 0;


}